### PR TITLE
Clickable cards

### PR DESCRIPTION
**Pull request by RealMahmoud**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -44,18 +44,18 @@
 
             <div class="col-sm-12 col-md-12 col-lg-6 mb-3">
                 <div class="card border border-brown shadow">
-                    <div class="card-body text-center">
+                    <a class="card-body text-center" href="https://bscscan.com/token/0x0de08c1abe5fb86dd7fd2ac90400ace305138d5b" style="color: #4f4f4f;">
```
Comment (mbidenaio) [inline public/index.html:47]: Please use _global_variables.BSC_EXPLORER_ and _global_variables.BSC_CONTRACT_ here

File: public/index.html [hunk 2]
```diff
@@ -44,18 +44,18 @@
...
                         <h4><img src="/images/bsc.svg" style="height:40px;" class="mb-2 me-2">Total Wrapped iDNA issued</h4>
                         <h5 id="total-bsc">- iDNAs</h5>
-                    </div>
+                    </a>
                 </div>
             </div>
             <div class="col-sm-12 col-md-12 col-lg-6 mb-3">
                 <div class="card border border-brown shadow">
-                    <div class="card-body text-center">
+                    <a class="card-body text-center" href="https://scan.idena.io/address/0x98D16d7021930b788135dD834983394fF2De9869" style="color: #4f4f4f;">
```
Comment (mbidenaio) [inline public/index.html:55]: Please use _global_variables.IDENA_EXPLORER_ and _global_variables.IDENA_WALLET_ here

Comment (RealMahmoud) [reply public/index.html:55]: Ok will do

File: public/js/index.js [hunk 3]
```diff
@@ -1,7 +1,12 @@
 document.addEventListener('DOMContentLoaded', function () {
+    updateCardsLinks();
     fetchData();
 }, false);
 
+function updateCardsLinks() {
+    document.getElementById("bsc-link").href = "https://bscscan.com/token/" + global_variables.BSC_CONTRACT;
```
Comment (mbidenaio) [inline public/js/index.js:7]: Please use also _global_variables.BSC_EXPLORER_

File: public/js/index.js [hunk 4]
```diff
@@ -1,7 +1,12 @@
...
+    document.getElementById("idena-link").href = "https://scan.idena.io/address/" + global_variables.IDENA_WALLET;
```
Comment (mbidenaio) [inline public/js/index.js:8]: Please use also _global_variables.IDENA_EXPLORER_

File: public/js/index.js [hunk 5]
```diff
@@ -1,7 +1,12 @@
 document.addEventListener('DOMContentLoaded', function () {
+    updateCardsLinks();
     fetchData();
 }, false);
 
+function updateCardsLinks() {
+    document.getElementById("bsc-link").href = global_variables.BSC_EXPLORER + global_variables.BSC_CONTRACT;
```
Comment (mbidenaio) [inline public/js/index.js:7]: Please add _/token/_ after _global_variables.BSC_EXPLORER_

File: public/js/index.js [hunk 6]
```diff
@@ -1,7 +1,12 @@
...
+    document.getElementById("idena-link").href = global_variables.IDENA_EXPLORER + global_variables.IDENA_WALLET;
```
Comment (mbidenaio) [inline public/js/index.js:8]: Please add _/address/_ after _global_variables.IDENA_EXPLORER_

**Comment by RealMahmoud** [comment]
Sorry about that .

File: public/js/index.js [hunk 7]
```diff
@@ -1,7 +1,12 @@
 document.addEventListener('DOMContentLoaded', function () {
+    updateCardsLinks();
     fetchData();
 }, false);
 
+function updateCardsLinks() {
+    document.getElementById("bsc-link").href = global_variables.BSC_EXPLORER + "/token/" + global_variables.BSC_CONTRACT;
+    document.getElementById("idena-link").href = global_variables.IDENA_EXPLORER + "/address/ " + global_variables.IDENA_WALLET;
```
Comment (mbidenaio) [inline public/js/index.js:8]: Please fix _"/address/_ " -> _"/address/"_ :)

**Pull request merged**